install.rst

```diff
@@ -201,6 +202,51 @@ To install these, you can use:
   ``swig``, ``swig-python``, ``openssl``, ``bison``, and ``flex`` packages
   provide the required dependencies.
 
+* Windows
+
+  Compiling on Windows requires the installation of a development environment.
+  Zeek currently builds on Visual Studio 2019, and you can either install the
+  full version including the UI tools or you can install the command-line tools
+  and build from a shell. The instructions below describe how to install the
+  command-line tools, but are not necessary if you install the full VS2019
+  package. You will need to install Chocolatey_ in order to install the
+  dependencies as instructed below. It's possible to install them from other
+  sources, but that is left to the reader to determine how.
```

Comment:
```suggestion
  sources, which we leave to the reader.
```
Could perhaps also add that feedback on such alternatives is welcome?

Reply:
Added mention of msys2 and cygwin.